qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
That sounds good, I agree coming up with a better plugin registration API is a better use of time, and this solution is backwards-compatible. Can we rename to something a little more descriptive, such as `parameter_help_text` or `parameter_descriptions`?